NAS-102881 rework help text docs. doc'ed dragability of boxes (#1469)
freenas/freenas-docs: userguide/intro.rst

* Right-click help dialog has been added to the :ref:`Shell`.

> * :ref:`Help text <help text>` boxes can be pinned and moved on the

Reviewer: I'm not really sure what pinning does now.  Maybe the last part should be "... pinned in place."?
```suggestion
* :ref:`Help text <help text>` boxes can be moved or pinned.
```

>   screen.

Reviewer: ```suggestion
```
